docs/developer-docs/latest/update-migration-guides/migration-guides/v4/code/backend/policies.md

```diff
@@ -0,0 +1,125 @@
+
+::: strapi v3/v4 comparison
+
+In both Strapi v3 and v4, policies handle authorization. Policies can be global or scoped (i.e. applied to a specific API or plugin) and can be applied to REST controller's actions or to GraphQL resolvers.
+
+In Strapi v3, policies are Koa middlewares accepting or rejecting requests based on the REST context. As middlewares, v3 policies always receive a Koa context, either coming from the REST request or built from the GraphQL resolver arguments.
+
+In Strapi v4, [policies](/developer-docs/latest/development/backend-customization/policies.md#policies) are functions that should return `true` or `undefined` for the request to be accepted. v4 policies receive a custom context based on where the policy has been called from (e.g. a REST controller’s action or a GraphQL resolver).
+:::
+
+To migrate a policy to Strapi v4:
+
+1. Move the policy file in the appropriate folder, depending on the policy scope:
+  
+    | Policy scope      | Folder                                                                                                                                                                                                           |
+    | ----------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+    | Global            | `./src/policies`                                                                                                                                                                                                 |
+    | A specific API    | `./src/api/<api-name>/policies`                                                                                                                                                                                  |
+    | A specific plugin | `./src/plugins/<plugin-name>/policies`<br/><br/>(see [plugin policies migration](/developer-docs/latest/update-migration-guides/migration-guides/v4/plugin/update-folder-structure.md#moving-policies) documentation) |
+
+2. (_optional_) Update the policy code. Strapi v4 policies are functions returning `true` or `undefined` to authorize the request.
+
+::: note
+In Strapi v4, depending on whether the policy is applied to REST or GraphQL, the function has access to different contexts:
+
+- Both REST and GraphQL contexts have access to the `is(type: string): boolean`, `type: string`, and `state: object` attributes.
+- The GraphQL context has access to `parent`, `args`, `context`, `info` & `http` (which contains the Koa context) (see [GraphQL customization](http://localhost:8080/developer-docs/latest/plugins/graphql.html#custom-configuration-for-resolvers) documentation).
+- With REST requests, the controller context is merged with the policy context.
```

Comment:
Uhhhhh not sure on this, pretty sure the policy context is thrown away after the policy is successfully passed.